### PR TITLE
Fix SASL required snippet

### PR DESCRIPTION
**Pull request by linuxdaemon**

### CODE REVIEW

File: docs/3/configuration/useful-snippets.md [hunk 1]
```diff
@@ -29,12 +29,10 @@ To do this you create an allow connect class which only applies pre-connection a
 You can also combine this with other `<connect>` fields such as adding `port="6660-6669"` to only apply to plain text connections or `webirc="*"` to only apply to connections via WebIRC gateways (requires [the cgiirc module](/3/modules/cgiirc)).
 
 ```xml
-<connect allow="*"
```
Comment (genius3000) [inline docs/3/configuration/useful-snippets.md:32]: This connect block needs to remain (treat this example snippet as the only connect classes in the config).

File: docs/3/configuration/useful-snippets.md [hunk 2]
```diff
@@ -29,12 +29,10 @@ To do this you create an allow connect class which only applies pre-connection a
...
-         registered="no"
-         requireaccount="no">
+<connect allow="*" requireaccount="yes">
```
Comment (genius3000) [inline docs/3/configuration/useful-snippets.md:32]: Space this out to match the others (incl. newlines). Also add the 'registered="yes"' to remain explicit and demonstrate exactly what's going on (I know it's technically redundant).

File: docs/3/configuration/useful-snippets.md (hunk 1 above)
Comment (linuxdaemon) [reply docs/3/configuration/useful-snippets.md:32]: updated

**Comment by genius3000** [comment]
Thanks for your contribution! I pushed an additional edit and re-ordered the fields to match the base configuration examples (importance then alphabetical ordering) before merging.

**Pull request merged**